packages/pymedphys_pinnacle/src/pymedphys_pinnacle/exporttool/pinnacle.py

```diff
@@ -0,0 +1,158 @@
+# RTSTRUCT, RTPLAN or RTDOSE.
+class Pinnacle:
+
+    logger = None  # Logger to use for all logging
+
+    path = ""  # Path to the root directory of the dataset (containing
+    # the Patient file)
+    patient_info = None  # The patient data read from
+    plans = None  # Pinnacle plans for this path
+    images = None  # Images found in image.info
+
+    def __init__(self, path, logger=None):
+
+        self.path = path
+        self.logger = logger
+
+        if not self.logger:
+            self.logger = logging.getLogger(__name__)
+            self.logger.setLevel(logging.DEBUG)
+
+            if len(self.logger.handlers) == 0:
+                ch = logging.StreamHandler(sys.stdout)
+                formatter = logging.Formatter(
+                    '%(asctime)s - %(levelname)s - %(message)s')
+                ch.setFormatter(formatter)
+                ch.setLevel(logging.DEBUG)
+
+                self.logger.addHandler(ch)
+
+    def get_patient_info(self):
+
+        if not self.patient_info:
+            path_patient = os.path.join(self.path, "Patient")
+            self.logger.debug(
+                'Reading patient data from: {0}'.format(path_patient))
+            self.patient_info = pinn_to_dict(path_patient)
+
+            # Set the full patient name
+            self.patient_info['FullName'] = "{0}^{1}^{2}^".format(
+                self.patient_info['LastName'],
+                self.patient_info['FirstName'],
+                self.patient_info['MiddleName'])
+
+            # gets birthday string with numbers and dashes
+            dobstr = self.patient_info['DateOfBirth']
+            if '-' in dobstr:
+                dob_list = dobstr.split('-')
+            elif '/' in dobstr:
+                dob_list = dobstr.split('/')
+            else:
+                dob_list = dobstr.split(' ')
+
+            dob = ""
+            for num in dob_list:
+                if len(num) == 1:
+                    num = '0' + num
+                dob = dob + num
+
+            self.patient_info['DOB'] = dob
+
+        return self.patient_info
+
+    def get_plans(self):
+
+        # Read patient info to populate patients plns
+        if not self.plans:
+            self.get_patient_info()
+
+            self.plans = []
+            for plan in self.patient_info['PlanList']:
+                path_plan = os.path.join(
+                    self.path, "Plan_"+str(plan['PlanID']))
+                self.plans.append(PinnaclePlan(self, path_plan, plan))
+
+        return self.plans
+
+    def get_images(self):
+
+        # Read patient info to populate patients images
+        if not self.images:
+            self.get_patient_info()
+
+            self.images = []
+            for image in self.patient_info['ImageSetList']:
+                pi = PinnacleImage(self, self.path, image)
+                self.images.append(pi)
+
+        return self.images
+
+    def export_struct(self, plan, export_path="."):
```

Comment:
If we change the class to `PinnacleExport` I imagine the `export` could be removed from here.